index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="style.css">
+  <title>My Portfolio</title>
+</head>
+
+<body>
+  <main>
+    <header>
+      <nav class="container">
+        <a href="#">My logo</a>
+        <img class="menu" src="./images/Icon - Menu.png" alt="Menu Icon">
+      </nav>
+      <section>

```

Comment:
- [ ] please remember the header tag is used as a page header and the section tag represents a generic standalone section of the document. Please remove the section tag, the header tag.